### PR TITLE
Change maintainer to label maintainer in Dockerfile

### LINKED ISSUE #245: Replace MAINTAINER with LABEL MAINTAINER
The MAINTAINER instruction has been [deprecated](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated)

difficulty/newcomer

### PR DESCRIPTION
**Pull request by raju249**
Dockerfile: Change maintainer to label maintainer

This PR contains updated Dockerfile as per updated [doc](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated)
Changed `MAINTAINER` to `LABEL maintainer`

Closes #245

### CODE REVIEW

**Review by gureks — CHANGES_REQUESTED** [review]
Please read the Writing Good Commits guide here: http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html
and change the shortlog and body accordingly

**Comment by jayvdb** [comment]
This repo is broken at the moment, so this wont be merged until that is fixed.

However, please fix your commit message.

**Comment by raju249** [comment]
I have updated the commit message as per the guidelines.
Sorry that I missed it earlier.

Now, since this is blocked and won't be merged ..should I move on for `difficulty/low` issue or wait till this PR is merged ?

**Review by prnvdixit — CHANGES_REQUESTED** [review]
@raju249 
1. Please, mention issue reference in your ```Commit``` (you have mentioned it only in your PR description yet) :wink:
2. The length of Shortlog is more than that is specified https://coala.io/git 😄
3. You should mention the reasoning of changes in the commit description too - "This ensures that deprecation of "MAINTAINER" is handled." or something descriptive like this.

**Comment by prnvdixit** [comment]
Also, I personally will recommend that you should move to difficulty/low issues only after this PR is atleast approved ("ack"ed) - If not merged, so that you have a better understanding of things to be kept in mind in general workflow and you get more reviews related to your code (than commit) on your low-PR. 👍

**Comment by raju249** [comment]
Thanks @prnvdixit 
Refactoring this commit 😃

File: Dockerfile
Comment (RaiVaibhav) [inline Dockerfile:2]: There have to some way to get the things directly by viewing it, here you did't give any difference between `persons` mail and his `name`, that's why we use `<>`.
Simply replace `"Fabian Neuschmidt fabian@neuschmidt.de"` to 
`"Fabian Neuschmidt <fabian@neuschmidt.de>"` will be more helpful, although your changes will work totally fine as we are not bounded here, we can set any `key` and his `value`.

**Review by RaiVaibhav — CHANGES_REQUESTED** [review]

**Comment by duskybomb** [comment]
@Makman2 assign this to me.

**Comment by Makman2** [comment]
Closing due to inactivity.

**Pull request closed without merging**
